topgun/k8s/rebalance_test.go

```diff
@@ -14,22 +13,21 @@ import (
 
 var _ = Describe("Worker Rebalancing", func() {
 	var (
-		releaseName  string
-		namespace    string
 		proxySession *gexec.Session
 		atcEndpoint  string
 	)
 
 	BeforeEach(func() {
-		releaseName = fmt.Sprintf("topgun-wr-%d", randomGenerator.Int())
-		namespace = releaseName
+		setReleaseNameAndNamespace("wr")
 
 		deployConcourseChart(releaseName,
 			"--set=concourse.worker.ephemeral=true",
 			"--set=worker.replicas=1",
 			"--set=web.replicas=2",
 			"--set=concourse.worker.rebalanceInterval=5s",
-			"--set=concourse.worker.baggageclaim.driver=detect")
+			"--set=concourse.worker.baggageclaim.driver=detect",
+			"--set=web.env[0].name=CONCOURSE_TSA_PEER_ADDRESS",
```

Comment:
Hey, why is this one needed? 

Is it only to circumvent the fact that we have new flags to be configured?

If that's the case, I'd suggest instead updating [`refs.json` in `concourse/charts`](https://github.com/concourse/charts/blob/maintenance/refs.json) to point to a branch where we can live a [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) to get the new flags for `5.1` in and then let the automation take care of producing the "merged view" of "upstream + new flags" 🤔

Reply:
@cirocosta 
Added a PR for updating the Helm Chart with the flag changes. Removed these temp changes from this PR.

Reply:
Great! Thanks!!